ports/pixman/portfile.cmake

```
@@ -27,6 +27,12 @@ elseif(VCPKG_TARGET_ARCHITECTURE MATCHES "arm")
                -Dsse2=disabled
                -Dssse3=disabled
        )
+elseif(VCPKG_TARGET_ARCHITECTURE MATCHES "mips")
+    list(APPEND OPTIONS
+            -Dmmx=disabled
+            -Dsse2=disabled
+            -Dssse3=disabled
+    )
```

```suggestion
               -Dssse3=disabled)
elseif(VCPKG_TARGET_ARCHITECTURE MATCHES "mips")
    list(APPEND OPTIONS
            -Dmmx=disabled
            -Dsse2=disabled
            -Dssse3=disabled
        )
```
Just a suggestion, keep a consistent writing style.

---

: > Got it.